### DISCUSSION TITLE
Just one save which i started directly with Expedition 1

### DISCUSSION THREAD

**Discussion by Samiral1981** [leodium/leodium-ReLive-NoMansSky-Expeditions-Offline#46] [Q&A] [answered]
How can i end the Expedition and keep playing with all the inventory, base, ship, mutitool and frigate? Is there a way to convert the save into my main save when i finished the Expedition? How i end it in first place when i have no other "Main Save".  do i have to Start all over again with normal save? Anyone know how that works?

**Comment by cwmonkey** [comment, marked as answer]
@Samiral1981 If you started the expedition from the main screen, your expedition save gets converted to a main save when the expedition is over and you keep everything on that save.

If you start the expedition on the Anomaly from the terminal, you get a chance to send items back to your main save as well as copy over your multitool and ship.

Also, the things you unlock will be available to ALL of your saves at the QuickSilver vendor on the Anomaly.

**Reply by Samiral1981** [reply]
@cwmonkey Thats good to know! Thank you for the info and for making it possible to play all the old Expeditions. Best regards!
